pkg/controller/applicationmonitoring/applicationmonitoring_controller.go

```diff
@@ -280,19 +282,21 @@ func (r *ReconcileApplicationMonitoring) CreateResource(cr *applicationmonitorin
 	return resource, nil
 }
 
-func (r *ReconcileApplicationMonitoring) GetHostFromRoute(object runtime.Object) (string, error) {
-	if object == nil {
-		return "", errors.New("Error getting host from route: runtime object was nil")
+func (r *ReconcileApplicationMonitoring) GetHostFromRoute(namespacedName types.NamespacedName) (string, error) {
```

Comment:
I think this could be changed to `getHostFromRoute` rather than `GetHostFromRoute`. Would you like me to change that here?

Reply:
Yes makes sense, this should not be exported.

Reply:
Cool, I'll modify the commit to change that now :+1:

Reply:
Done!